### PR TITLE
UX: Multichain: Ensure percentage of account name always shown

### LINKED ISSUE #18969: Token value cut off when using long account name + long token name
## Bug description
Combination of really long account name and long token name cuts off value for token

See second account:

### PR DESCRIPTION
**Pull request by darkwing**
## Explanation

At present the CSS structure of the new account picker allows the value cell to completely overtake the token cell.  This PR ensures there's a minimum value for the amount of token cell that's shown.

Fixes: https://github.com/MetaMask/metamask-extension/issues/18969

## Screenshots/Screencaps

SCR-20230515-kqcn

## Pre-merge author checklist

- [ ] I've clearly explained:
  - [ ] What problem this PR is solving
  - [ ] How this problem was solved
  - [ ] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

**Review by georgewrmarshall — DISMISSED** [review]
LGTM! Nice work. Not sure if a connected account may need to be adjusted

Screenshot 2023-05-09 at 5 51 03 PM

**Comment by SaraCheikh** [comment]
Hi @darkwing, I did some testing and these max widths would work in design for the left (without accounting with the Account avatar, ) and right content: 
Screenshot 2023-05-09 at 15 33 24
Screenshot 2023-05-09 at 15 33 09

Let me know if this would work for you too.

**Comment by SaraCheikh** [comment]
I did some  testing in Storybook with varying max and mini widths for the account name and balance  in order to minimize truncation, and this is the best combination I could find:
Screenshot 2023-05-10 at 22 50 07

It would def help to decrease the font size of the titles (Account Name and Balance) from our current 16px medium to 14px medium, I think this new size is closer to our existing font size in the account list. @georgewrmarshall @garrettbear @Akatori-Design Is it possible for us to override this value specifically for the account ? We wouldn't be maintaining  the pattern of using the same typography token for titles in cell components, but it would help us  prevent cases where both the account name and the token value are truncated.

**Comment by SaraCheikh** [comment]
@darkwing values for account not connected:
Screenshot 2023-05-11 at 16 00 03

**Comment by georgewrmarshall** [comment]
It would be great if could find ways of fitting the information without always turning to decreasing the font-size. It reduces accessibility and I imagine this is largely what caused many of our inconsistencies that we currently have in the extension? I understand finding a solution may take more time and resources than we can manage at this time so if reducing the font-size will unblock this then I suppose that's ok but we should make sure that this is reflected in all AccountCell components across all platforms and update in Figma as well

**Review by georgewrmarshall — COMMENTED** [review]
Looks good but I don't think we want to use static font-size because it could cause inconsistencies my suggestion would be to refactor the CurrencyDisplay component to use the `Text` component and invert control using a prop object

File: ui/components/multichain/account-list-item/index.scss [hunk 1]
```diff
@@ -30,11 +53,21 @@
...
+    overflow: hidden;
     display: inline;
+
+    &__prefix {
+      padding-right: unset;
+    }
+
+    &__suffix {
+      font-size: 12px !important;
+    }
```
[the remark this answers is not in the document]
Comment (georgewrmarshall) [reply ui/components/multichain/account-list-item/index.scss:71]: Agreed

**Review by georgewrmarshall — APPROVED** [review]
LGTM! 🚀

**Review by NidhiKJha — APPROVED** [review]

**Pull request merged**